ls1intum/Artemis
# `Plagiarsim checks`: Fix default value of after due date checks

### Checklist
#### General
#### Client

### Motivation and Context
Fixes #7651
Fixes #7611
Follow-up of #7665

### Description
When creating or importing an exercise, the "run plagiarism checks" checkbox was active, even if cpc was disabled. While this had no direct effect, the UI is confusing and should be clear in showing what options are active when working with an exercise.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a new Exercise -> Check that CPC and the checks after the due date are disabled by default.
3. Activate CPC -> see that the checks also get enabled
4. Deactivate CPC -> see that the checks automatically get disabled
5. Check that creating the exercise works

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress


#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage
#### Client

| Class/File | Line Coverage | Confirmation (assert/expect) |
|------------|--------------:|---------------------:|
| [exercise-update-plagiarism.component.ts](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5778/latest/artifact/shared/Coverage-Report-Client-Tests/app/exercises/shared/plagiarism/exercise-update-plagiarism/exercise-update-plagiarism.component.ts.html) | 100% | ✅ |
